package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "cdk-watchful",
-  "version": "0.5.0",
-  "description": "Watching your CDK apps since 2019",
+  "name": "cdk-watchful-edge",
+  "version": "0.6.0",
+  "description": "Watching your CDK apps since 2019 - Edge",
```

Comment:
Why the name change?

Reply:
@eladb Apologies for the erroneous PR!  I was attempting to settle the versioning issues with the latest release of CDK without impacting the official release workflow.  If you're open to a PR to update to 1.46.0, I'll open one without the name change.

Reply:
Absolutely

Reply:
@eladb https://github.com/eladb/cdk-watchful/pull/20